dynamic_instrumentation/src/point/build.rs

```diff
@@ -139,33 +140,33 @@ impl<'tcx> InstrumentationBuilder<'_, 'tcx> {
         self
     }
 
-    pub fn debug_mir(mut self, loc: Location) -> Self {
-        self.point.metadata.debug = {
-            let block = &self.body.basic_blocks()[loc.block];
-            if loc.statement_index == block.statements.len() {
-                match &block.terminator().kind {
-                    TerminatorKind::Call {
-                        args,
-                        destination,
-                        func,
-                        ..
-                    } if destination.is_some() => {
-                        let mut s = format!("{:?} = {:?}(", destination.unwrap().0, func);
-                        for (i, arg) in args.iter().enumerate() {
-                            if i > 0 {
-                                s.push_str(", ");
-                            }
-                            s.push_str(&format!("{:?}", arg));
-                        }
-                        s.push(')');
-                        s
-                    }
-                    _ => String::from(""),
-                }
-            } else {
-                format!("{:?}", block.statements[loc.statement_index])
+    fn debug_mir_to_string(&self, loc: Location) -> String {
+        let block = &self.body.basic_blocks()[loc.block];
+        if loc.statement_index != block.statements.len() {
+            return format!("{:?}", block.statements[loc.statement_index]);
+        }
+        match &block.terminator().kind {
+            TerminatorKind::Call {
+                args,
+                destination: Some((destination, _)),
```

Comment:
This was an easy fix compared to the `match ... if destination.is_some()` and `destination.unwrap()`.